.gitignore

```diff
@@ -4,3 +4,6 @@ __pycache__
 .spyproject
 /.idea
 .idea
+*.jpg
+*.png
+*.h264
```

Comment:
Could this be a separate pull request, please? I know it's a bit annoying, but it's not really related to the rest of this pull request, could be merged earlier, and would leave less stuff to review here. Thanks!